### ISSUE TITLE
Use torch.tanh and torch.sigmoid

### ISSUE THREAD

**Issue by tridao** [bharathgs/NALU#3]
As of Pytorch 0.4.1, torch.nn.functional.sigmoid and torch.nn.functional.tanh are deprecated in favor of torch.sigmoid and torch.tanh.

**Comment by bharathgs** [comment]
has been updated.
